package/com.unity.formats.usd/Runtime/Scripts/IO/Materials/StandardShaderExporter.cs

```diff
@@ -419,7 +415,7 @@ public class StandardShaderExporter : ShaderExporterBase
             if (mat.HasProperty("_OcclusionMap") && mat.GetTexture("_OcclusionMap") != null)
             {
                 var newTex = SetupTexture(scene, usdShaderPath, mat, surface, Vector4.one, destTexturePath,
-                    "_OcclusionMap", "r");
+                    "_OcclusionMap", "g");
```

Comment:
Can we add a comment here (and a link to docs) about why we use the "g" channel for occlusion.

Reply:
done